isis/src/mro/apps/hical/hical.cpp

```diff
@@ -217,11 +199,42 @@ namespace Isis {
                            "in your configuration file.";
             throw IException(IException::User, mess, _FILEINFO_);
           }
-          ZeroDarkRate zdr(hiconf);
-          calVars->add(hiconf.getProfileName(), zdr.ref());
-          ZdrHist = zdr.History();
-          if ( hiprof.exists("DumpModuleFile") ) {
-            zdr.Dump(hiconf.getMatrixSource("DumpModuleFile",hiprof));
+          try{
+            ZeroDarkRate zdr(hiconf);
+            calVars->add(hiconf.getProfileName(), zdr.ref());
+            ZdrHist = zdr.History();
+            if ( hiprof.exists("DumpModuleFile") ) {
+              zdr.Dump(hiconf.getMatrixSource("DumpModuleFile",hiprof));
+            }
+          }
+          catch(IException e){
+            if (hiprof.exists("Fallback")){
+              if (IsTrueValue(hiprof, "Fallback")){
+                zdrFallback = true;
+                calVars->add(hiconf.getProfileName(), HiVector(nsamps, 0.0));
+                std::cout << "Falling back to ZeroDark implementation. Unable to initialize ZeroDarkRate "
+                          << "module with the following error:" << std::endl << e.what() << "\nContinuing..."<< std::endl;
```

Comment:
A warning might still want to be printed via cerr. Or maybe [qWarning](https://doc.qt.io/qt-6/qtglobal.html#qWarning)()?